docs/application/dotnet/guides/applications/uiapplication/application_lifecycle.md

```diff
@@ -0,0 +1,186 @@
+# Application lifecycle
```

Comment:
When I opened this topic using staging link, I am not seeing breakdown of TOC structure in the left navigation bar for this topic. It was working fine earlier. Please check and update.

Reply:
Fixed.

Reply:
I am still seeing the same issue. Please see attached screenshot for reference. After opening this topic/file using staging link, the left navigation bar doesn't display the exact location of the topic in the tree structure.

![Tree Struture Error](https://user-images.githubusercontent.com/85727367/135254243-7c33786d-fbc2-4779-8bf9-148dc5a9e8c3.PNG)